invoicing_app/tests/tests.py

```diff
@@ -85,13 +87,15 @@ def test_handle_with_no_country(self):
             ['AR', 'BR']
         )
 
+    # Only for the old script
```

Comment:
It may be worth it to split this test case class into two: one for the old command, and one for the new one. This way we won't have to rely on comments to tell tests apart, and we also get the added benefit that tests for each command will be able to run separately.

In general, comments quickly turn unmaintainable and out of sync with code. On the other hand, a test case class won't, since it is a code construct itself.

Reply:
Ok! Thanks! I'll do it.